util/desktop/desktop.go

```diff
@@ -0,0 +1,54 @@
+package desktop
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+)
+
+var (
+	bbEnabledOnce sync.Once
+	bbEnabled     bool
+)
+
+func BuildBackendEnabled() bool {
+	bbEnabledOnce.Do(func() {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return
+		}
+		_, err = os.Stat(filepath.Join(home, ".docker", "desktop-build", ".lastaccess"))
+		bbEnabled = err == nil
+	})
+	return bbEnabled
+}
+
+type BuildDetailsOpt struct {
+	Builder string
+	Node    string
+	Ref     string
+}
+
+func BuildDetailsOutput(opts map[string]BuildDetailsOpt) string {
+	var out bytes.Buffer
+	out.WriteString("Open build details on Docker Desktop: ")
```

Comment:
`s/on/in/`

Maybe `s/Open/Review/` . The shorter we can make this prefix the better.